docs/2fa/users/guide.md

```diff
@@ -0,0 +1,100 @@
+
+![](../../assets/2fa/setup-1.png)
+
+![](../../assets/2fa/setup-2.png)
+
+![](../../assets/2fa/setup-3.png)
+
+And we also support 1password for the 2FA secret representation. You need to prepare a **Login** item in 1password, configure the basic information, then you can use **Scan QR Code** function in 1password. It will make it more flexiable and efficient.
+
+![](../../assets/2fa/1password.png)
+
+Recommend configuration in 1password
+
+| | |
+| --- | --- |
+| username | wallet address |
+| website | https://app.2fa.guru |
+| tags | the target network |
+
+- Once you successfully sign up, we recommend you add a recovery wallet for your primary account, which can be used to reset your 2FA device. Otherwise, if you lost your 2FA device, then the only thing you can do is to disable the 2FA protection before you use your 2FA protected assets.
+
+  You need to send a bind transaction to our 2FA management contract after our authentication, which means the gas fee is required.
+
+  There is a binding window for each account after signup, you must bind the recovery wallet in **7 days**, otherwise you cannot bind anymore.
+
+![](../../assets/2fa/bind-1.png)
+
+![](../../assets/2fa/bind-2.png)
+
+---
+
+## Reset 2FA devices
+
+If you lost your 2FA device unfortunately, you can reset it if you have been bind a recovery wallet, the reset processes are quite similar as setup's:
+
+<!-- - Visit [Automata 2FA Guru](https://app.2fa.guru). -->
+- Visit Automata 2FA Guru.
+
+- Use metamask and connect your recovery wallet.
+
+- Click **Reset 2FA** button in **Wallet Actions**, input your primary wallet address and sign the message. If it's matched, then you can scan the new secret to reset the 2FA device. There is no transaction on chain, so it's free to reset a 2FA device. And the recovery wallet is not required to sign up for 2FA.
+
+![](../../assets/2fa/reset-1.png)
+
+![](../../assets/2fa/reset-2.png)
+
+---
+
+## Make authorised transfer
+
+Once you sign up 2FA for your wallet, you will benefit from all 2FA protected assets and dApps. If you get any 2FA protected assets, you will be able to transfer them in **Manage my Wallet Assets** panel.
+
+It will check your 2FA code first and return you a valid signature, which will be used in the transfer transaction.
+
+![](../../assets/2fa/transfer-1.png)
+
+![](../../assets/2fa/transfer-2.png)
+
+![](../../assets/2fa/transfer-3.png)
+
+---
+
+## Disable 2FA protection
+
+If you lost your 2FA device and don't bind any recovery wallet, or you don't want to use 2FA anymore, you can disable 2FA protection in 2FA dApp. There is a cooldown period when disabling 2FA protection, you can cancel the disable 2FA protection request before this window. Attention, the cooldown period is **7 Days**, and you cannot use **2FA protection** anymore.
```

Comment:
If you've lost your 2FA device and no respective recovery wallet has been bound, you always have the option to disable 2FA protection in the 2FA dApp. There is a cooldown period for disabling 2FA protection, and you can cancel the disable 2FA protection request before this period ends. The cooldown period is **7 Days**, and after which you would not be able to use **2FA protection** anymore.